docs/setup/rke2-cluster.md

```diff
@@ -37,28 +37,12 @@ EOF
 virsh net-update egress add-last ip-dhcp-host host.xml --live
 ```
 
-4. Create an XML file containing the following
-
-```shell
-cat << EOF > ~/vbmc/dns.xml
-<host ip='192.168.125.100'>
-  <hostname>media.suse.baremetal</hostname>
-</host>
-EOF
-```
-
-5. Live update the egress network once again
-
-```shell
-virsh net-update egress add-last dns-host dns.xml --live
-```
-
-6. SSH into the metal3-core VM
+4. SSH into the metal3-core VM
 ```shell
 ssh metal@192.168.125.99
 ```
 
-7. Download the example manifests
+5. Download the example manifests
 
 ```shell
 curl https://raw.githubusercontent.com/suse-edge/metal3-demo/main/docs/example-manifests/dhcp/rke2-control-plane.yaml > rke2-control-plane.yaml
```

Comment:
Currently, this is not good because you can't test local changes of the example-manifests yamls, Can we copy these files into the VM and apply them instead of downloading the main branch each time?

Reply:
It's completely fine to do this in a different PR

Reply:
Actually things become much simpler if we just run helm and apply manifests from the hypervisor host, so I've pushed changes which do that to the branch now.